### PR TITLE
Add Twingate

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Twingate is a secure access platform that enables businesses to connect their users to internal resources and applications without relying on traditional VPNs. It provides a modern, Zero Trust security model, ensuring secure and seamless access to corporate networks for remote and hybrid workforces. Twingate offers 2FA for the account login verification process. The website is ranked 98,640 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=twingate.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: img/t/twingate.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26 26"><path d="M0 25V0h115v25Z"/><path fill="#F7F8F9" d="M5.5 10.6v10.9l3.7-2.6v-5.7c0-2.9.9-3.9 3.3-5.6.8-.6 2-1.4 3-2.3V0L8.9 5c-2.4 1.8-3.3 2.7-3.3 5.6Zm5 14.4V14.2c0-3 .8-4 3.2-5.7l6.8-5v10.3c0 2.6 0 4-2.7 6-2.2 1.7-7.4 5.2-7.4 5.2Z"/></svg>
```
Comment (Carlgo11) [inline img/t/twingate.com.svg]: Hmmm something looks odd about the logo. Try this one :thinking: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28" style="background:#000;fill:#fff"><path d="M8 12.52v8.68s1.3-.89 2.75-1.91v-4.77c0-2.35.7-3.1 2.62-4.53l2.62-1.95V4l-5.37 3.99C8.71 9.41 8 10.17 8 12.52ZM12 24v-8.68c0-2.35.7-3.1 2.62-4.53l5.37-3.99V15c.02 2.1.03 3.22-2.11 4.84A226.84 226.84 0 0 1 12 24"/></svg>
```

File: entries/t/twingate.com.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "Twingate": {
+    "domain": "twingate.com",
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Biometrics",
+      "Yubico OTP"
```
Comment (Carlgo11) [inline entries/t/twingate.com.json]: What's described in the documentation sounds a lot like FIDO2 U2F.
As a rule of thumb, if Windows Hello or "Security Keys" are mentioned, it is most likely U2F.
```suggestion
      "u2f"
```

File: entries/t/twingate.com.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    ],
+    "documentation": "https://www.twingate.com/docs/two-factor-authentication-security-policies",
+    "categories": [
+      "remote"
```
Comment (Carlgo11) [inline entries/t/twingate.com.json:10]: I think security may be a more appropriate category. Or perhaps both
```suggestion
      "security"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/t/twingate.com.svg (hunk 1 above)
Comment (Yang-Nankai) [reply img/t/twingate.com.svg]: Thanks!

File: entries/t/twingate.com.json (hunk 2 above)
Comment (Yang-Nankai) [reply entries/t/twingate.com.json]: I see. Also, do biometric features such as fingerprints not count? @Carlgo11

File: entries/t/twingate.com.json (hunk 3 above)
Comment (Yang-Nankai) [reply entries/t/twingate.com.json:10]: I feel a little confused about the classification here, but I personally prefer remote. ❤️ @Carlgo11

Comment (Carlgo11) [reply entries/t/twingate.com.json:10]: Okay! Let's do remote then.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**